### PR TITLE
1 second isn't always enough for Travis

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

File: cromwellApiClient/src/test/scala/cromwell/api/CromwellResponseFailedSpec.scala [hunk 1]
```diff
@@ -15,7 +15,7 @@ import scala.language.postfixOps
 
 class CromwellResponseFailedSpec extends TestKit(ActorSystem()) with AsyncFlatSpecLike with Matchers with BeforeAndAfterAll {
   override def afterAll(): Unit = {
-    Await.ready(system.terminate(), 1 second)
+    Await.ready(system.terminate(), 10 second)
```
Comment (mcovarr) [inline cromwellApiClient/src/test/scala/cromwell/api/CromwellResponseFailedSpec.scala]: plural

Comment (geoffjentry) [inline cromwellApiClient/src/test/scala/cromwell/api/CromwellResponseFailedSpec.scala]: Could time dilation be useful here?

Comment (cjllanwarne) [reply cromwellApiClient/src/test/scala/cromwell/api/CromwellResponseFailedSpec.scala]: Perhaps, what would that look like/mean?

**Comment by geoffjentry** [comment]
:+1:

**Comment by katevoss** [comment]
Is this related to #2401 ?

**Comment by cjllanwarne** [comment]
@katevoss nope, this is just making a single test more reliable

**Pull request merged**